### PR TITLE
Finish adding edit activity

### LINKED ISSUE #131: [DEV] As a user, I want to update an existing activity

### PR DESCRIPTION
**Pull request by simonjulianl**
### Summary

- Resolves #131 

### Detail

- Update all test cases and command to update an existing activity 

### Pictures / Gif

image

### Checks

- [x] Link PR to issue
- [x] Add Reviewer
- [x] Pass CI

### CODE REVIEW

File: src/main/java/gomedic/model/activity/UniqueActivityList.java [hunk 1]
```diff
@@ -58,6 +58,30 @@ public int getNewActivityId() {
         throw new MaxAddressBookCapacityReached();
     }
 
+    /**
+     * Replaces the activity {@code target} in the list with {@code editedActivity}.
+     * {@code target} must exist in the list.
+     * The person identity of {@code editedActivity} must not be the same as another existing activity in the list.
```
Comment (radiankrisno) [inline src/main/java/gomedic/model/activity/UniqueActivityList.java:64]: should it be activity here?

**Review by radiankrisno — CHANGES_REQUESTED** [review]
Almost LGTM, nitpick

File: src/main/java/gomedic/model/activity/UniqueActivityList.java (hunk 1 above)
Comment (simonjulianl) [reply src/main/java/gomedic/model/activity/UniqueActivityList.java]: yup

**Review by radiankrisno — APPROVED** [review]
LGTM

**Pull request merged**